### ISSUE TITLE
Accessibility: The luminosity contrast ratio of white text on top of camera screen is less than 4.5:1. 

### ISSUE THREAD

**Issue by malvikam** [journeyapps/zxing-android-embedded#552]
#### Description of the problem:
We use the barcodescanner library in our app. The background of the barcodescanner camera screen is black with 60% transparency. Any white text on top of the screen with transparent background is not meeting the accessibility criteria of contrast ratio of at least 4.5:1. Can the opacity of background be increased to meet accessibility critera https://www.w3.org/TR/WCAG20-TECHS/G18.html? 
Fix in the library will improve accessibility of all apps using this library.

**Which library version are you using?** E.g. 3.2.0.
3.3.0

**Which phone/tablet are you using, and which Android version does it run? (e.g. Samsung Galaxy S5,
  Android 5.0)**

**Does the same happen on other devices or an emulator?**

**Can you reproduce the issue in the sample project included with the library? If not, can you
  provide your own sample project or sample code that produces this error?**

**In the case of an error do you have a stack trace or adb logs?**
